lib/Backend/Lower.cpp

```diff
@@ -9783,6 +9783,14 @@ Lowerer::LowerEqualityBranch(IR::Instr* instr, IR::JnHelperMethod helper)
     }
     if (!needHelper)
     {
+        if (instr->AsBranchInstr()->GetTarget()->m_isLoopTop)
```

Comment:
>f (instr->AsBranchInstr()->GetTarget()->m_isLoopTop) [](start = 9, length = 52)

Even if the target label is loop top, how do you know that the branch is a back edge and not a forward branch?

Reply:
Oops, you are right. I can add an m_number comparison.